data/json/items/gunmod/sights.json

```diff
@@ -185,7 +185,7 @@
     "id": "tele_sight",
     "type": "GUNMOD",
     "name": "telescopic sight",
-    "description": "A simple telescopic sight, essentially a small telescope with crosshairs.  Increases weight but improves accuracy.",
+    "description": "A simple hand-crafted telescopic sight, essentially a small telescope with crosshairs. Not as good as the ones made before the cataclysm.  Increases weight but improves accuracy.",
```

Comment:
```suggestion
    "description": "A simple hand-crafted telescopic sight, essentially a small telescope with crosshairs.  Not as good as the ones made before the cataclysm.  Increases weight but improves accuracy.",
```

Reply:
Is it just me or is calling it 'the cataclysm' a little 4th wall breaking? I find it pretty jarring, I'd go for "before the dead rose" or something.

Reply:
We have plenty mentions of the cataclysm.

Reply:
Why would there be 2 spaces after a period?

Reply:
That's convention in the game. I believe it's due to monospaced font style rules, which require two spaces after a period.